dcsg-bugsnag-build-resource.yml

```diff
@@ -0,0 +1,5 @@
+name: bugsnag-build-version
```

Comment:
we usually use the name as the name a user would use in their pipeline. I see from the github page that you expect `bugsnag-build` without the `-version`. I don't mind merging the PR this way, but just thought to ask in case it is not intentional.

Reply:
I will change 👍